File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        String[] speaks = {"aya","ye","woo","ma"};
        int answer = 0;
        for(String babble: babbling){
            int babbleLen = babble.length();
            int babSum = 0;
            for(String speak: speaks){
                if(babble.contains(speak)){
                    babSum += speak.length();
                }
            }
            if(babbleLen == babSum)
                answer++;
        }
        return answer;
    }
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    babble_list = ["aya","ye","woo","ma"]
    answer = 0
    for babble in babbling:
        babble_len = len(babble)
        comp = sum(len(idx) for idx in babble_list if idx in babble)
        if babble_len == comp:
            answer+= 1
    return answer